src/AstrometryTransform.cc

```diff
@@ -733,6 +805,60 @@ void AstrometryTransformPolynomial::transformPosAndErrors(FatPoint const &in, Fa
     out = res;
 }
 
+void AstrometryTransformPolynomial::transformPosAndErrors(FatPoint const &in, FatPoint &out) const {
+    switch (_nterms) {
+        case 3:
+        {
+            Eigen::Matrix<double, 3, 1> monomials;
+            _transformPosAndErrorsImpl<1>(monomials, in, out);
+            break;
+        }
+        case 6:
+        {
+            Eigen::Matrix<double, 6, 1> monomials;
+            _transformPosAndErrorsImpl<2>(monomials, in, out);
+            break;
+        }
+        case 10:
+        {
+            Eigen::Matrix<double, 10, 1> monomials;
+            _transformPosAndErrorsImpl<3>(monomials, in, out);
+            break;
+        }
+        case 15:
+        {
+            Eigen::Matrix<double, 15, 1> monomials;
+            _transformPosAndErrorsImpl<4>(monomials, in, out);
+            break;
+        }
+        case 21:
+        {
+            Eigen::Matrix<double, 21, 1> monomials;
+            _transformPosAndErrorsImpl<5>(monomials, in, out);
+            break;
+        }
+        case 28:
+        {
+            Eigen::Matrix<double, 28, 1> monomials;
+            _transformPosAndErrorsImpl<6>(monomials, in, out);
+            break;
+        }
+        case 36:
+        {
+            Eigen::Matrix<double, 36, 1> monomials;
+            _transformPosAndErrorsImpl<7>(monomials, in, out);
+            break;
+        }
+        default:
```

Comment:
Should we stop at 7th order, or go to 9th, in case we want to use that high an order?